contrib/kubespray/script/validate_layout_schema.py

```diff
@@ -79,6 +85,11 @@ def check_layout(layout):
     if 'pai-worker' in masters[0] and masters[0]['pai-worker'] == 'true':
         logger.error("One machine can not be pai-master and pai-worker at the same time.")
         return False
+    # if cluster_config.enable_hived_scheduler is false, there should be <= 1 type of computing device
+    if 'enable_hived_scheduler' in cluster_config and cluster_config['enable_hived_scheduler'] is False:
```

Comment:
Will hived scheduler be enabled if `enable_hived_scheduler` is not set in `config.yaml` ?

Reply:
yes